src/Akka.Persistence.Sql/Query/SqlReadJournal.cs

```diff
@@ -59,6 +59,11 @@ public class SqlReadJournal :
             _system = system;
 
             var connFact = new AkkaPersistenceDataConnectionFactory(_readJournalConfig);
+            
+            // Fix for https://github.com/akkadotnet/Akka.Persistence.Sql/issues/344
+            var writeJournal = Persistence.Instance.Apply(system).JournalFor(writePluginId);
```

Comment:
Start the `SqlWriteJournal` if it has not been started already - technically the `_eventAdapters = Persistence.Instance.Apply(system).AdaptersFor(writePluginId);` also does this implicitly.